src/main/interface/msp.c

```diff
@@ -768,10 +768,12 @@ bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst) {
     case MSP_NAME: {
         // Show warning for DJI OSD instead of pilot name
         // works if osd warnings enabled, osd_warn_dji is on and usb is not connected
+        #ifdef USE_OSD
```

Comment:
This should be okay, however, if the FC does not have an analog OSD chip, it still technically could perform DJI OSD functions. But DJI custom osd would not work without enabling USE_OSD. I feel like theres a small user base that would lose DJI OSD on these FCs

Reply:
interesting. i don't know how to resolve such.